packages/messaging/android/build.gradle

```diff
@@ -33,11 +32,10 @@ android {
 repositories {
     google()
     mavenCentral()
-    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
 }
 
 dependencies {
     api "com.adobe.marketing.mobile:optimize:2.+"
```

Comment:
the optimize extension is no longer a dependency for messaging and can be removed.